sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -37,6 +37,22 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Default port range for dynamic source port allocation
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_MIN (49152U)
+#define GNRC_SOCK_DYN_PORTRANGE_MAX (65535U)
+#define GNRC_SOCK_DYN_PORTRANGE_NUM (GNRC_SOCK_DYN_PORTRANGE_MAX - GNRC_SOCK_DYN_PORTRANGE_MIN + 1)
+#define GNRC_SOCK_DYN_PORTRANGE_ERR (0)
+
+/**
+ * @brief   Offset for next dynamic port
+ *
+ * Currently set to a static (prime) offset, but could be random, too
+ * see https://tools.ietf.org/html/rfc6056#section-3.3.3
+ */
+#define GNRC_SOCK_DYN_PORTRANGE_OFF (17U)
```

Comment:
the RFC actually says it has to be a random component, see code comment. However,  I felt it might not be good to have the overhead of random by default in RIOT